Add 3 entries to the `/knowledge` center (#1917)
vercel/docs: pages/knowledge/a-record-and-caa-with-vercel.mdx

import Layout from '~/components/layout/knowledge'
import Note from '~/components/text/note'

export const meta = {

>   title: 'Can I use my domain on Vercel with A records?',

Reviewer: ```suggestion
  title: 'Can I Use My Domain on Vercel with a Records?',
```

>   description: 'Information about how to use A records with Vercel.',

Reviewer: ```suggestion
  description: 'Information on how to use A records with Vercel to verify a domain.',
```

  editUrl: 'pages/knowledge/a-record-and-caa-with-vercel.mdx',
  url: '/knowledge/a-record-and-caa-with-vercel',

>   published: '2020-06-15T13:46:54.155Z',

Reviewer: ```suggestion
  published: '2020-06-17T13:46:54.155Z',
```

  lastEdited: '2020-06-15T13:46:54.155Z'
}

> Vercel is currently testing the [Anycast methodology](https://en.wikipedia.org/wiki/Anycast) for DNS and routing.

Reviewer: ```suggestion
Vercel is currently testing the [Anycast methodology](https://en.wikipedia.org/wiki/Anycast) for DNS and routing. This allows those using a custom domain to verify it without requiring a change in nameservers or to add a CNAME record.
```

> 

Reviewer: ```suggestion

## Verifying with A Records

```

> The following records are required:

Reviewer: ```suggestion
To verify the domain, add the following DNS records with your domain provider:
```

> - Add an A record with the value `76.76.21.21`.

Reviewer: ```suggestion
- An A record with the value `76.76.21.21`.
```

> - Add a CAA record on the root of your domain with the value `"0 issue letsencrypt.org"`.

Reviewer: ```suggestion
- A CAA record on the root of your domain with the value `"0 issue letsencrypt.org"`.
```

> If you followed the instructions above successfully, your domain should be available for use on any Vercel project that you own.

Reviewer: ```suggestion
If you followed the instructions above successfully, your domain will be available for use on any Vercel project that you own. All that's required is to add it to the project you wish to use it with.
```
